components/openindiana/release/files/release

```diff
@@ -1,4 +1,4 @@
-             OpenIndiana Hipster 2019.10 (powered by illumos)
+             OpenIndiana Hipster %%DISTRIBUTION_VERSION%% (powered by illumos)
```

Comment:
Note, first line in motd file format is used at least by installer to determine URL to release notes, by openindiana-welcome to get distribution version and perhaps by some other software, so we can't just change it to '$(DISTRIBUTION_NAME) $(DISTRIBUTION_VERSION)'